### PR TITLE
Sb3/fix compile javadoc

### PR DESCRIPTION
**Pull request by JoergSiebahn**

### CODE REVIEW

File: sda-commons-cloudevents/src/main/java/org/sdase/commons/spring/boot/cloudevents/CloudEventV1.java [hunk 1]
```diff
@@ -23,48 +23,56 @@
...
 @JsonClassDescription(
-    "This is a manifestation of the [CloudEvents](https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/spec.md) "
-        + "specification. The field documentation will contain the official documentation and also "
-        + "some hints how values should should be set when used on the SDA platform.")
+    """
+    This is a manifestation of the
+    [CloudEvents](https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/spec.md)
+    specification.
+    The field documentation will contain the official documentation and also some hints how values
+    should should be set when used on the SDA platform.
```
Comment (preethivenkatesh-sdase) [inline sda-commons-cloudevents/src/main/java/org/sdase/commons/spring/boot/cloudevents/CloudEventV1.java:31]:
```suggestion
    should be set when used on the SDA platform.
```

File: sda-commons-cloudevents/src/main/java/org/sdase/commons/spring/boot/cloudevents/CloudEventV1.java [hunk 2]
```diff
@@ -23,48 +23,56 @@
...
-          + "unique for each distinct event. If a duplicate event is re-sent (e.g. due to a "
-          + "network error) it MAY have the same id. Consumers MAY assume that Events with "
-          + "identical `source` and `id` are duplicates.\n"
-          + "\n"
-          + "**SDA**: The default is to use a random UUID.")
+      """
+      Identifies the event. Producers MUST ensure that `source` + `id` is unique for each distinct
+      event.
+      If a duplicate event is re-sent (e.g. due to a network error) it MAY have the same id.
+      Consumers MAY assume that Events with identical `source` and `id` are duplicates.
```
Comment (preethivenkatesh-sdase) [inline sda-commons-cloudevents/src/main/java/org/sdase/commons/spring/boot/cloudevents/CloudEventV1.java:40]: The uppercase usage for **must** and **may** is intentional, or?

File: sda-commons-starter-kafka/src/main/java/org/sdase/commons/spring/boot/kafka/SdaKafkaConsumerConfiguration.java [hunk 3]
```diff
@@ -123,6 +123,9 @@ public CommonErrorHandler containerStoppingErrorHandler() {
    * original topic name suffixed with .DLT) and to the same partition as the original record.
    * Therefore, when you use the default resolver, the dead-letter topic must have at least as many
    * partitions as the original topic.
+   *
+   * @return the default error handler that retries according the {@link
```
Comment (preethivenkatesh-sdase) [inline sda-commons-starter-kafka/src/main/java/org/sdase/commons/spring/boot/kafka/SdaKafkaConsumerConfiguration.java:127]:
```suggestion
   * @return the default error handler that retries according to the {@link
```

File: sda-commons-cloudevents/src/main/java/org/sdase/commons/spring/boot/cloudevents/CloudEventV1.java (hunk 2 above)
Comment (JoergSiebahn) [reply sda-commons-cloudevents/src/main/java/org/sdase/commons/spring/boot/cloudevents/CloudEventV1.java:40]: Yes. Most of docs are copied from [the original spec](https://github.com/cloudevents/spec/blob/main/cloudevents/spec.md#notational-conventions) (maybe an older variant).

**Review by preethivenkatesh-sdase — APPROVED** [review]

**Pull request merged**